### ISSUE TITLE
v15.0-rc1 Live editing mode not working for blocklist and blockgrid

### ISSUE THREAD

**Issue by kasparboelkjeldsen** [umbraco/Umbraco-CMS#17274]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

15.0-rc1

### Bug summary

Making a change in the editor for a blocklist or blockgrid block, does not trigger UMB_PROPERTY_CONTEXT or UMB_BLOCK_ENTRY_CONTEXT and no change is observed until you hit the Update button, even though Live editing mode is enabled for the datatype.

### Specifics

This works in v14

```typescript
this.consumeContext(UMB_BLOCK_GRID_ENTRY_CONTEXT, (context) => {
      context.contentValues().then((content) => {
        this.observe(content, (content) => {
          console.log('content', content);
        });
      });
    });
```

I would expect this console.log to get hit when making a change (ie. writing a piece of text in a textbox or what have you) when editing a block. But nothing happens until you hit the update button.

### Steps to reproduce

Create a custom view for a blockgrid or list, and consume contexts like in the example above.

### Expected result / actual result

I would expect, observing either all the content in a block, or the value of the entiry property that -something- should update when I make a change, after having enabled live editing.

---
_This item has been added to our backlog AB#45805_

**Comment by kasparboelkjeldsen** [comment]
@iOvergaard @nielslyngsoe sorry but this still seems to be an issue in rc3.

```js
this.consumeContext(UMB_PROPERTY_CONTEXT, (propertyContext) => {
      this.observe(propertyContext.value, (value) => {
        console.log('this should fire when I make a live change?')
      });
});
....
this.consumeContext(UMB_BLOCK_ENTRY_CONTEXT, async (context) => {
    context.contentValues().then(async (blockContent) => {
        this.observe(blockContent, async (content) => {
          console.log('this should change when I edit my block no?')
        });
      });
});
        ```
        
        These only fire after I hit the submit button on the blockgrid modal

**Comment by nielslyngsoe** [comment]
Hi @kasparboelkjeldsen Thanks for reporting, I can also confirm that it does not work in 15.next.
We will make sure it does, thanks again.

**Comment by asawyermarathon** [comment]
This may be the same / similar issue: I just noticed that listening for settingsValues() changes never fires either in 15.0.0:

blockEntryContext.settingsValues().then(async (blockSettings) => {
    console.log("settingsValues", blockSettings);
    blockEntryContext.observe(blockSettings, async (content) => {
        console.log('this should change when I edit my block setttings no?')
    });
});

**Comment by madsrasmussen** [comment]
Fixed by: #17578
